docs/src/model_creation/examples/basic_CRN_library.md

```diff
@@ -115,12 +115,12 @@ using Plots
 oplt = plot(osol; title = "Reaction rate equation (ODE)")
 splt = plot(ssol; title = "Chemical Langevin equation (SDE)")
 jplt = plot(jsol; title = "Stochastic chemical kinetics (Jump)")
-plot(oplt, splt, jplt; lw = 2, size=(800,800), layout = (3,1)) 
-plot!(bottom_margin = 3Plots.Measures.mm) # hide
```

Comment:
Without this doesn’t the text get cutoff?

Reply:
Yesadding it back in